### PR TITLE
youtube-cleanup: Add warning for `remove-channel-info` in video description

### PR DESCRIPTION
**Pull request by JohnyP36**
See discussion #503 

Is this a possible solution or should we add this warning at the button of the template?

### CODE REVIEW

File: data/filters/templates/youtube-cleanup.yaml [hunk 1]
```diff
@@ -24,7 +24,9 @@ params:
     type: checkbox
     default: false
   - name: remove-channel-info
-    description: Removes channel information below the video description
+    description: | 
+      Removes channel information below the video description 
+      Warning: in newer YT layouts enabling this rule may result in braking down the functionality of clicking on the information icon in the right-top corner of the video
```
Comment (JohnyP36) [inline data/filters/templates/youtube-cleanup.yaml:29]: Maybe it is possible to put this text in *italics*...?

**Comment by xvello** [comment]
Heya @JohnyP36! The `description` field is single-line only and designed for short text.

We could add the warning at the bottom of the template description (the markdown part) but BTH I don't think it's worth posting a warning. If you're that concerned about the button not doing anything, is there an easy way to hide that button?

**Comment by JohnyP36** [comment]
We could add the rule `www.youtube.com##.ytp-cards-button-icon-default > .ytp-cards-button-icon`  
but I think that we can just leave it as how it is now, as fore some YT layouts it is still functioning. 

So, that means I will close this PR if you agree.

**Comment by xvello** [comment]
Let's not overthink it indeed. Closing the PR then, but let's discuss if there's actual reports of people being confused by it.

**Pull request closed without merging**